package.json

```
@@ -2,6 +2,7 @@
   "name": "@playwright/mcp",
   "version": "0.0.46",
   "description": "Playwright Tools for MCP",
+  "mcpName": "com.microsoft/playwright",
```

we probably want this called `playwright-mcp`. we already have a different MCP server distributed as part of our test runner, and we don't want to namesquat ourselves for future potential products.
```suggestion
  "mcpName": "com.microsoft/playwright-mcp",
```

---

The name here is for the MCP Registry, so I think the `-mcp` suffix would be redundant. If this is not the "main" Playwright MCP server, then I think a different qualifier would make sense. This is like suffixing an npm package name with `npm` or `package`. But it's up to you.

---

(to be clear the `title` will be used for display here, not the `name`, but I noticed the `names` are generally not peppered with `mcp` suffixes -- just providing you more context to make the decision you think is best)

---

Done.